test/unit_tests/braket/default_simulator/test_statevector_simulation.py

```diff
@@ -0,0 +1,276 @@
+# Copyright 2019-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can you also test the behavior of attempting to simulate a circuit with noise?

Reply:
when when using state vector simulator with noise, it throws TypeError. Example: `TypeError: instruction BitFlip is not supported by DefaultSimulator`